gslib/addlhelp/metadata.py

```diff
@@ -45,146 +45,34 @@
-  instance) applying another level of compression via Content-Encoding is
-  typically detrimental to both object size and performance and should be
-  avoided.
-
-  Note also that gsutil provides an easy way to cause content to be compressed
-  and stored with Content-Encoding: gzip: see the -z and -Z options in
-  "gsutil help cp".
-
-
-<B>CONTENT-DISPOSITION</B>
-  You can set Content-Disposition on your objects, to specify presentation
-  information about the data being transmitted. Here's an example uploading
-  files from a local directory named ``attachments``:
-
-    gsutil -h 'Content-Disposition:attachment; filename=filename.ext' \\
-           cp -r attachments gs://bucket/attachments
-
-  Setting the Content-Disposition allows you to control presentation style
-  of the content, for example determining whether an attachment should be
-  automatically displayed vs should require some form of action from the user to
-  open it.  See https://tools.ietf.org/html/rfc6266
-  for more details about the meaning of Content-Disposition.
-
-
-<B>CUSTOM METADATA</B>
-  You can add your own custom metadata (e.g,. for use by your application)
-  to a Google Cloud Storage object by using "x-goog-meta" with -h. For example:
-
-    gsutil -h x-goog-meta-reviewer:jane cp mycode.java gs://bucket/reviews
-
-  You can add multiple differently-named custom metadata fields to each object.
-
+    the ``gsutil setmeta`` command. See "gsutil help setmeta".
 
 <B>SETTABLE FIELDS; FIELD VALUES</B>
   You can't set some metadata fields, such as ETag and Content-Length. The
   fields you can set are:
 
-  - Cache-Control
-  - Content-Disposition
-  - Content-Encoding
-  - Content-Language
-  - Content-Type
+  - ``Cache-Control``
+  - ``Content-Disposition``
+  - ``Content-Encoding``
+  - ``Content-Language``
+  - ``Content-Type``
+  - ``Custom-Time``
   - Custom metadata
 
   Field names are case-insensitive.
 
   All fields and their values must consist only of ASCII characters, with the
-  exception of values for x-goog-meta- fields, which may contain arbitrary
+  exception of values for ``x-goog-meta-`` fields, which may contain arbitrary
   Unicode values. Note that when setting metadata using the XML API, which sends
-  custom metadata as HTTP headers, Unicode characters will be encoded using
+  custom metadata as HTTP headers, Unicode characters are encoded using
   UTF-8, then url-encoded to ASCII. For example:
 
     gsutil setmeta -h "x-goog-meta-foo: ã" gs://bucket/object
 
-  would store the custom metadata key-value pair of "foo" and "%C3%A3".
-  Subsequently, running "ls -L" using the JSON API to list the object's metadata
-  would print "%C3%A3", while "ls -L" using the XML API would url-decode this
-  value automatically, printing the character "ã".
+  stores the custom metadata key-value pair of ``foo`` and ``%C3%A3``.
+  Subsequently, running ``ls -` using the JSON API to list the object's metadata
```

Comment:
Missing closing backtick?

Reply:
Also missing a capital "L". This was likely a "highlight-delete" accident while converting HTML <code> elements to backticks.

Good catch!